packages/airnode-examples/scripts/run-airnode-locally.ts

```diff
@@ -8,10 +8,11 @@ const main = async () => {
     return;
   }
 
+  // Check if we're executing in Github CI - if we are, run the airnode-client development image
+  const imageName = process.env.CI ? `api3/airnode-client-dev:${process.env.GITHUB_SHA}` : `api3/airnode-client:latest`;
```

Comment:
@aquarat - 
1) you beat me to the suggestion of splitting up the docker build steps 😄
2) I'm not sure this is working correctly- the E2E examples step in the latest master commit took [7m 43s](https://github.com/api3dao/airnode/runs/5459250423?check_suite_focus=true) and built the images as part of the step, while the E2E examples step in this commit took [8m 21s](https://github.com/api3dao/airnode/runs/5460950565?check_suite_focus=true). It should be much faster with a prebuilt image.

Reply:
Disregard- it looks like you pushed a commit solving this while I was reviewing

Reply:
Hey, yes, sorry, I've fully focused on optimising this now - I should have done this sooner.
The last total run took ~14 minutes, the biggest contributor were the unit tests, followed by e2e tests->node flow. If the airnode-examples e2e test is split from the matrix (so that the e2e matrix no longer relies on the docker build) we'll get back down to ~11 minutes :)

Reply:
With the test split out we're down to ~12 minutes: https://github.com/api3dao/airnode/actions/runs/1950318787

Reply:
Nice!